hs_core/hydroshare/hs_bagit.py

```diff
@@ -138,6 +138,16 @@ def create_bag_files(resource):
     for f in resFiles:
         a.add_resource(f)
 
+    # in current implementation, collection has no resource files
+    if resource.resource_type == "CollectionResource" and resource.metadata.collection:
+        for contained_res in resource.metadata.collection.resources.all():
+            contained_res_id = contained_res.short_id
+            landing_page = '{hs_url}/resource/{res_id}/'.format(hs_url=current_site_url, res_id=contained_res_id)
```

Comment:
@zhiyuli Are you sure this url needs to point to the resource landing page and not the resource bag?